gitlab/v4/objects.py

```diff
@@ -2641,6 +2637,43 @@ class ProjectAccessRequestManager(ListMixin, CreateMixin, DeleteMixin,
     _from_parent_attrs = {'project_id': 'id'}
 
 
+class ProjectMergeRequestApproval(SaveMixin, RESTObject):
+    _id_attr = None
+
+
+class ProjectMergeRequestApprovalManager(GetWithoutIdMixin, UpdateMixin,
+                                         RESTManager):
+    _path = '/projects/%(project_id)s/approvals'
+    _obj_cls = ProjectMergeRequestApproval
+    _from_parent_attrs = {'project_id': 'id'}
+    _update_attrs = (tuple(),
+                     ('approvals_before_merge', 'reset_approvals_on_push',
+                      'disable_overriding_approvers_per_merge_request'))
+    _update_post = True
+
+    @exc.on_http_error(exc.GitlabUpdateError)
+    def change_approvers(self, approver_ids=[], approver_group_ids=[],
```

Comment:
Could you rename this method `set_approvers` ?

Reply:
Done.